src/LocalBackend/LocalLibrary.vala

```diff
@@ -198,6 +198,8 @@ public class Noise.LocalLibrary : Library {
 
             set_music_folder_thread.begin (folder);
         }
+        
+        Settings.Main.get_default ().music_folder = folder;
```

Comment:
I think it's better to save inside the `if` above.

Reply:
Ok I made the changes in the next commit. It seems the Travis CI build failed though and I'm not sure why

Reply:
Travis fails due to some connection problem with ppa.launchpad.net, I will close the PR and reopen it to trigger it to build again.